qpc/messages.py

```diff
@@ -233,7 +234,10 @@
 BECOME_PASSWORD = 'Provide a privilege escalation password to be used when '\
     'running a network scan.'
 
-
+JOB_ID_NOT_FOUND = 'Merge report job "%s" not found.'
+JOB_ID_STATUS = 'Job id: %s status is %s'
+DISPLAY_REPORT_ID = \
+    'To view the report run: qpc report summary --report "%s" --csv.'
```

Comment:
This might be better as:
 Created merge report with id: "%s". To view, run qpc report summary --report "%s" --csv

But I will leave that up to you